### PR TITLE
Upgrade gramine from v1.2 to v1.3.1

### PR DESCRIPTION
**Pull request by billionairiam**
## Description of the PR 
Upgrade gramine from v1.2 to v1.3.1

### CODE REVIEW

File: cczoo/horizontal_fl/image_classification/Makefile [hunk 1]
```diff
@@ -18,6 +18,8 @@
 ARCH_LIBDIR ?= /lib/$(shell $(CC) -dumpmachine)
 # ENTRYPOINT ?= $(realpath $(shell sh -c "command -v python3"))
 
+RA_TYPE ?= none
```
Comment (guzongmin) [inline cczoo/horizontal_fl/image_classification/Makefile:21]: Should we set the default value with "dcap"?

File: cczoo/horizontal_fl/recommendation_system/Makefile [hunk 2]
```diff
@@ -18,6 +18,8 @@
 ARCH_LIBDIR ?= /lib/$(shell $(CC) -dumpmachine)
 # ENTRYPOINT ?= $(realpath $(shell sh -c "command -v python3"))
 
+RA_TYPE ?= none
```
Comment (guzongmin) [inline cczoo/horizontal_fl/recommendation_system/Makefile:21]: Should we set the default value with "dcap"?

Comment (billionairiam) [reply cczoo/horizontal_fl/recommendation_system/Makefile:21]: OK, will modify it.

**Review by guzongmin — APPROVED** [review]
LGTM

**Pull request merged**